package.json

```diff
@@ -199,7 +199,7 @@
     "rollup-plugin-replace": "^2.1.0",
     "rollup-plugin-uglify": "^6.0.1",
     "sass-loader": "^7.1.0",
-    "semantic-release": "^16.0.0-beta.19",
+    "semantic-release": "^16.0.0-beta.39",
```

Comment:
Why update the semantic-release deps? These can be a bit unstable as we're running the beta versions.

Reply:
this was to try and fix that weird semantic-release issue we had where it just lost the previous tag